examples/nidcpower_source_dc_voltage/NIDCPowerSourceDCVoltageMultiSite.pinmap

```diff
@@ -7,17 +7,13 @@
 	</Instruments>
 	<Pins>
 		<DUTPin name="Pin1" />
-		<DUTPin name="Pin2" />
 	</Pins>
 	<PinGroups></PinGroups>
 	<Sites>
 		<Site siteNumber="0" />
 		<Site siteNumber="1" />
 	</Sites>
 	<Connections>
-		<Connection pin="Pin1" siteNumber="0" instrument="DCPower1" channel="0" />
-		<Connection pin="Pin1" siteNumber="1" instrument="DCPower1" channel="2" />
-		<Connection pin="Pin2" siteNumber="0" instrument="DCPower1" channel="1" />
-		<Connection pin="Pin2" siteNumber="1" instrument="DCPower1" channel="3" />
+		<Connection pin="Pin1" siteNumber="0,1" instrument="DCPower1" channel="0" />
```

Comment:
Please revert the multi-site pinmap. I would like it to demonstrate how to return data for multiple pins and sites, and using one channel for both sites doesn't really demonstrate that.

Add a note to README.md that `NIDCPowerSourceDCVoltageMultiSite.pinmap` requires an SMU with 4 or more channels.

Reply:
Got it. Reverted changes made in `NIDCPowerSourceDCVoltageMultiSite.pinmap` and added a note in `Readme.md` as well.